tests/unit/authQueryStringTest.php

```diff
@@ -58,7 +58,13 @@ public function testGenerateSignature()
 
         $expected_to_sign = "POST\n$request_path\nauth_key=$auth_key&auth_timestamp=$time&auth_version=$auth_version&name=an_event";
         $expected_auth_signature = hash_hmac('sha256', $expected_to_sign, $auth_secret, false);
-        $expected_query_string = "auth_key=$auth_key&auth_signature=$expected_auth_signature&auth_timestamp=$time&auth_version=$auth_version&name=an_event";
+        $expected_query_string = [
```

Comment:
We could go with:
```suggestion
        $expected_query_params = [
```
and also change the function name.